.github/workflows/stack-regression-tests-ol.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches:
       - main
 jobs:
-   stack-tests:
```

Comment:
Was your intention here to create two jobs one called ol-setup and another called ol-tests to separate setup and test actions? Or you just wanted to change the name to differentiate ol vs wl test runs? If the latter, perhaps just pick a name and prefix it with ol-/wl- here and in stack-regression-tests-wl.yml. That way it is easier to tell them apart.